ajax/libs/sinon.js/package.json

```diff
@@ -16,6 +16,15 @@
     "type": "git",
     "url": "https://github.com/sinonjs/sinon"
   },
+  "npmName": "sinon",
+  "npmFileMap": [
+    {
+      "basePath": "pkg",
+      "files": [
+        "**/+(sinon.|sinon-no-sourcemaps.)*"
```

Comment:
@extend1994 did you mean that there are two options that we can choose?
First, is using auto-update but the `basePath` should be `""` (but why can't we use `pkg`, if there are no `dist` or `build` and it seem that the OK files always in `pkg` folder?)
Second, is just manually add old versions no need via auto-update mechanism?

I thought that if we set auto-update config after this PR merge, the auto-update mechanism will start to add the versions which not in CDNJS from the library. Isn't it? (https://github.com/cdnjs/cdnjs/pull/11439#issuecomment-312471744)

Or would you suggest the good way can add these versions which CDNJS didn't host?

Thank you.

cc @maruilian11 @pvnr0082t @PeterDaveHello